pkg/api/api.go

```diff
@@ -127,6 +140,15 @@ func (a API) ReceiveEvent(w http.ResponseWriter, r *http.Request) {
 	ctx := r.Context()
 	defer r.Body.Close()
 
+	// If self hosting and keys are not defined, error.
+	if a.requireKeys && len(a.localEventKeys) == 0 {
+		a.writeResponse(w, apiResponse{
+			StatusCode: http.StatusServiceUnavailable,
+			Error:      "Event keys are required to ingest events securely",
```

Comment:
Should we avoid using "ingest" in our product? Seems like it might cause confusion

Reply:
Good point - maybe we can change this to "send events" or similar in a future release cc @jpwilliams